### PR TITLE
[RW-8919][risk=no] Update AoU staging hostname

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:
AoU is updating hostname: [https://all-of-us-rw-staging.appspot.com`](https://all-of-us-rw-staging.appspot.com%60/)

to new host name: staging.fake-research-aou.org

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: e2e/resources/workbench-config.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ const test: IEnvConfig = {
 
 // workbench staging environment
 const staging: IEnvConfig = {
-  LOGIN_URL_DOMAIN_NAME: process.env.STAGING_LOGIN_URL || 'https://all-of-us-rw-staging.appspot.com',
+  LOGIN_URL_DOMAIN_NAME: process.env.STAGING_LOGIN_URL || 'https://staging.fake-research-aou.org',
   API_HOSTNAME: process.env.API_HOSTNAME || 'api-dot-all-of-us-rw-staging.appspot.com',
```
Comment (jmthibault79) [inline e2e/resources/workbench-config.ts:61]: Does the API_HOSTNAME need to change as well?

File: api/config/config_staging.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
   },
   "server": {
     "apiBaseUrl": "https:\/\/api-dot-all-of-us-rw-staging.appspot.com",
```
Comment (jmthibault79) [inline api/config/config_staging.json:71]: does apiBaseUrl also need to change?

Comment (jmthibault79) [reply api/config/config_staging.json:71]: Please also confirm these values:
* `:api_endpoint_host` in `environments.rb`
* `allOfUsApiUrl` in `ui/src/environments/environment.staging.ts`
* the `Content-Security-Policy-Report-Only` list in `ui/app.yaml`

File: e2e/resources/workbench-config.ts (hunk 1 above)
Comment (yonghaoy) [reply e2e/resources/workbench-config.ts:61]: ahh good point! I see we did change api server address

File: api/config/config_staging.json (hunk 2 above)
Comment (yonghaoy) [reply api/config/config_staging.json:71]: @jmthibault79 API is not changed for now. They may do it later.

**Review by jmthibault79 — APPROVED** [review]

File: api/config/config_staging.json (hunk 2 above)
Comment (jmthibault79) [reply api/config/config_staging.json:71]: The API change is #7056

**Pull request merged**